chapters/chapter-12/application/src/test/resources/localstack/init-ses.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/sh
 
-awslocal sqs create-queue --queue-name stratospheric-todo-sharing
-
 awslocal ses verify-email-identity --email-address noreply@stratospheric.dev
 awslocal ses verify-email-identity --email-address info@stratospheric.dev
 awslocal ses verify-email-identity --email-address tom@stratospheric.dev
 awslocal ses verify-email-identity --email-address bjoern@stratospheric.dev
 awslocal ses verify-email-identity --email-address philip@stratospheric.dev
 
-echo "Initialized."
+echo "SES Setup complete."
```

Comment:
I haven't tried, but I think that this change would make [AbstractDevIntegrationTest](https://github.com/stratospheric-dev/stratospheric/blob/4b64c791910e884883e1c39500d59fdcccd19388/application/src/test/java/dev/stratospheric/todoapp/AbstractDevIntegrationTest.java#L28) fail